OKR.md

```diff
@@ -0,0 +1,52 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q4
+
+**go-ipfs handles large datasets (1TB++) without a sweat**
+- `P0` - It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+- `P0` @hannahhoward - List a sharded directory with 1M entries over LAN in under 1 minute, with less than a second to the first entry.
+- `PX` - There is a prototype implementation of GraphSync for go-ipfs
+- `P0` @magik6k - There is a better and more performant datastore module (e.g Badger or better)
```

Comment:
@magik6k I can help with the Badger transition (or take ownership of this one if you want).

Reply:
Oh, sure, go ahead

Reply:
Thanks @schomatis, I'll update.